app/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+const bcrypt = require('bcryptjs');
+const { encryptionError } = require('../errors');
+const saltRounds = 10;
+exports.hashPassword = obj =>
```

Comment:
If this function only hash a string as a password why receives an object ?

Reply:
My idea was to create the first step of a kind of pipeline. hashPassword recieves the user object and returns the user with a hashed password.
I usually end up extracting all functions so I can have at the en of the module something like:
hashPassword(userData)
    .then(createUser)
    .then(sendResponse)
    .catch(sendError)

And then I can add or remove tasks. That was my initial idea thinking I might need to refactor in the future. But maybe this is more confusing than helping!

Reply:
Seems like a good idea to me now 😄